views/partials/navbar.ejs

```diff
@@ -1,6 +1,8 @@
 <header class="mb-auto">
   <nav id="nav" class="px-3 navbar navbar-expand-lg navbar-dark">
-    <a id="logo-title" class="navbar-brand text-warning contrast" href="/">G.O.O.N.</a>
+    <a id="logo-title" class="navbar-brand text-warning contrast" href="/"
```

Comment:
I'm ok with reformatting it to look prettier but the end of the bracket should stay on line 3 and </a> should all be on one line

Reply:
so unfortunately, prettier is what keeps changing it i think. every time i save the file, it formats it so that it goes back to a different line.

Reply:
Weird. We'll leave it alone I guess